web-registry/src/pages/CreditClassDetails/CreditClassDetailsSimple/CreditClassDetailsSimple.tsx

```diff
@@ -55,7 +60,21 @@ const CreditClassDetailsSimple: React.FC<
             }}
           >
             <Box sx={{ mb: 6 }}>
-              <Label size="sm" color="info.dark" mb={4}>
+              {image && (
+                <Box
+                  sx={{
+                    mb: { sm: 12.5 },
+                    mx: { xs: -4, sm: 0 },
+                  }}
+                >
+                  <img
+                    className={styles.image}
+                    alt={image.imageAlt || imageSrc}
+                    src={imageSrc}
+                  />
+                </Box>
```

Comment:
This could me merged I think:
```suggestion
                <Box 
                  component="img"
                  alt={image.imageAlt || imageSrc}
                  src={imageSrc}
                  sx={[
                  styles.image,
                  {
                    mb: { sm: 12.5 },
                    mx: { xs: -4, sm: 0 },
                  }
                  ]}
                />
```